tools/jdbc/src/main/java/org/duckdb/DuckDBArrowResult.java

```diff
@@ -0,0 +1,66 @@
+package org.duckdb;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@SuppressWarnings("rawtypes")
```

Comment:
Please use `Class<?>` instead of suppressing this warning

Reply:
Excellent did not know that one